dags/benchmark/pytorchxla_torchbench.py

```diff
@@ -14,37 +14,113 @@
 
 """A DAG to run all TorchBench tests with nightly version."""
 
-import datetime
 from airflow import models
+import datetime
 from configs import composer_env
 from configs.benchmark.pytorch import pytorchxla_torchbench_config as config
-from configs.vm_resource import TpuVersion, Zone
+from configs.vm_resource import *
```

Comment:
I think wildcard import this is not recommended? As we don't know where it comes from.